src/lib/chips/chip-list.ts

```diff
@@ -44,20 +67,32 @@ import {coerceBooleanProperty} from '@angular/cdk/coercion';
   exportAs: 'mdChipList',
   host: {
     '[attr.tabindex]': '_tabIndex',
+    '[attr.aria-describedby]': '_ariaDescribedby || null',
+    '[attr.aria-required]': 'required.toString()',
+    '[attr.aria-disabled]': 'disabled.toString()',
+    '[attr.aria-invalid]': 'errorState',
+    '[class.mat-chip-list-disabled]': 'disabled',
+    '[class.mat-chip-list-invalid]': 'errorState',
+    '[class.mat-chip-list-required]': 'required',
     'role': 'listbox',
     'class': 'mat-chip-list',
-
     '(focus)': 'focus()',
+    '(blur)': '_blur()',
     '(keydown)': '_keydown($event)'
   },
-  queries: {
-    chips: new ContentChildren(MdChip)
-  },
+  providers: [{provide: MdFormFieldControl, useExisting: MdChipList}],
   styleUrls: ['chips.css'],
   encapsulation: ViewEncapsulation.None,
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class MdChipList implements AfterContentInit, OnDestroy {
+export class MdChipList implements MdFormFieldControl<any>, ControlValueAccessor,
```

Comment:
Could you file an issue to track sharing common code for `role="listbox"` components in `cdk/listbox`? It would be good to capture what in particular would belong there